packages/driver/src/cy/ensures.coffee

```diff
@@ -27,7 +27,7 @@ create = (state, expect) ->
         ## if this is an element then ensure its currently attached
         ## to its document context
         if $dom.isElement(subject)
-          ensureAttached(subject, name)
+          ensureAttached(subject)
```

Comment:
this feels wrong... why was this removed? all the other ensures utilize this...?

Reply:
they do not, check out ensureNotReadOnly, ensureNotDisabled, and ensureVisibility